.circleci/scripts/upload_docker_image_to_s3.sh

```diff
@@ -0,0 +1,13 @@
+DOCKER_IMAGE_BASE_NAME=$(basename "${DOCKER_IMAGE}")
+export AWS_ACCESS_KEY_ID=${CIRCLECI_AWS_ACCESS_KEY_FOR_OSSCI_LINUX_BUILD_S3_BUCKET_V1}
+export AWS_SECRET_ACCESS_KEY=${CIRCLECI_AWS_SECRET_KEY_FOR_OSSCI_LINUX_BUILD_S3_BUCKET_V1}
+set +e
+retcode=$(aws s3 ls s3://ossci-linux-build/${DOCKER_IMAGE_TYPE}/${DOCKER_IMAGE_BASE_NAME}.tar)
```

Comment:
What's the access control on this S3 bucket? I hope it's not publicly writable...